### DISCUSSION TITLE
Any Hints to connect to Raspberry Pi zero

### DISCUSSION THREAD

**Discussion by cwoodffr** [AppGround-io/bluetooth-keyboard-and-mouse-support#219] [Q&A]
My phone pairs with the pi, but disconnects because there are no detected services.  When I do it from the app, it also fails.
I am controlling bluetooth services and pairing via SSH.  It pairs but won't stay connected due to lack of usable services. I need it to seek keyboard or input device capabilities.

**Comment by Staini** [comment]
I had some problems on my first attampts, too. But it should work.

Make your phone visible and start the pairing process from your pi.
Try it as fast as you can... because the pi/linux is very sensitive.
And I think it's important to accept the pairing code first on your phone and shortly after on your pi.
